packages/shared/src/components/cards/RaisedLabel.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import styles from './Card.module.css';
+import { SimpleTooltip } from '../tooltips/SimpleTooltip';
+
+export enum RaisedLabelType {
+  hot = 'Hot',
+  pinned = 'Pinned',
```

Comment:
For enums, we keep it using PascalCase, currently not part of the linter but hoping to be added soon. Then just probably update the use-cases.

```suggestion
  Hot = 'Hot',
  Pinned = 'Pinned',
```